ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor/Event.hs

```diff
@@ -86,22 +86,24 @@ data Event (muxMode :: MuxMode) peerAddr m a b
 
 
 --
--- STM transactions which detect 'Event's
+-- STM transactions which detect 'Event's (signals)
 --
 
 
+-- | A signal which returns an 'Event'.  Signals are combined together and
+-- passed used to fold the current state map.
+--
+type EventSignal (muxMode :: MuxMode) peerAddr m a b =
```

Comment:
Why Signal? I thought it had something to do with testing using the Signal API or something to do with time

Reply:
because it signals an event, nothing to do with the little testing framework.